### PR TITLE
feat(codes): update legal-jurisdiction and legal entitiy codes

### PR DESCRIPTION
**Pull request by sk91**
Enum cannot contain multiple equal values. So `Bonaire, Sint Eustatius and Saba` --` BQ` is one entity instead of 

```
"Caribbean Netherlands - Bonaire",
"Caribbean Netherlands - Sint Eustatius ",	
"Caribbean Netherlands - Saba ",
```

The standard used for codes is https://www.gleif.org/en/about-lei/code-lists/gleif-accepted-legal-jurisdictions-code-list

We can separate them back, but we will not comply anymore

--- for release, should be copied to `schema` folder

### CODE REVIEW

File: schema/attribute/legal-jurisdiction.json [hunk 1]
```diff
@@ -1,267 +1,520 @@
...
+		"United States - North Dakota",
+		"United States - Ohio",
+		"United States - Oklahoma",
+		"United States - Oregon",
+		"United States - Pennsylvania",
+		"United States - Rhode Island",
+		"United States - South Carolina",
+		"United States - South Dakota",
+		"United States - Tennessee",
+		"United States - Utah",
```
Comment (Chaviland7) [inline schema/attribute/legal-jurisdiction.json]: @sk91 it looks like you've left out `United States - Texas`. Was this on purpose?

Comment (sk91) [reply schema/attribute/legal-jurisdiction.json]: @Chaviland7 thous are the jurisdictions we had from FT.

@andregoncalves did we miss one?

Comment (andregoncalves) [reply schema/attribute/legal-jurisdiction.json]: I think we missed that. It's on the GLEIF legal jurisdictions codes, so it's a valid jurisdiction.
BTW I'm waiting for Marc's reply that list.

Comment (sk91) [reply schema/attribute/legal-jurisdiction.json]: fixed

**Pull request merged**